src/test/java/org/apache/sysds/test/functions/builtin/BuiltinALSDSTest.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.sysds.test.functions.builtin;
```

Comment:
the reason github complains about build is that you are missing a license here.

use `mvn package -P rat` to find out where the error is, it generates a file in /target/rat.txt that tells you.

Reply:
Done.